helper/test/odpthreads.c

```diff
@@ -160,6 +168,39 @@ int main(int argc, char *argv[])
 	if (ret < 0)
 		exit(EXIT_FAILURE);
 
+	/* Test threads with non-default stack size and sync timeout. */
+
+	pthread_attr_init(&attr);
+
+	if (pthread_attr_getstacksize(&attr, &stack_size)) {
+		printf("pthread_attr_getstacksize() failed\n");
+		return -1;
+	}
+
+	printf("\n");
+	printf("pthread default stack size is %zu\n", stack_size);
```

Comment:
The print values could be aligned for better readability. Now the output is a bit messy:
```
default cpu mask:           0xFFFF00000000
default num worker threads: 16
the first CPU:              32
new cpu mask:               0xFFFF00000000
new num worker threads:     16

Worker thread on CPU 34
Worker thread on CPU 35
Worker thread on CPU 41
Worker thread on CPU 42
Worker thread on CPU 32
Worker thread on CPU 37
Worker thread on CPU 40
Worker thread on CPU 38
Worker thread on CPU 36
Worker thread on CPU 33
Worker thread on CPU 39
Worker thread on CPU 43
Worker thread on CPU 44
Worker thread on CPU 45
Worker thread on CPU 47
Worker thread on CPU 46

pthread default stack size is 8388608
stack size soft limit (rlim_cur) is 8388608
use stack size 8384512
use sync timeout 5000000000
```